extensions/wikia/AnalyticsEngine/js/analytics_prod.js

```diff
@@ -141,6 +141,30 @@
 		['_setCustomVar', 5, 'LoginStatus',
 			!!window.wgUserName ? 'user' : 'anon', 3] );
 
+	/*
+	 * Remove when SOC-217 ABTest is finished
+	 */
+	/**
+	 * Get unconfirmed email AbTest user type
+	 * @returns {string}
+	 */
+	function getUnconfirmedEmailUserType() {
+		if ( !window.wgUserName ) {
+			return 'anon';
+		} else {
+			if ( window.wgNotConfirmedEmail === null ) {
```

Comment:
'if' conditions should not be whitespaced https://github.com/Wikia/guidelines/blob/master/JavaScript/CodingConventions.md#good-examples

Also, what do you think about a switch statement here?